Add What is Utreexo? episode (7 of 14)

edilmedeiros opened the pull request:
This is part of https://github.com/bitcointranscripts/bitcointranscripts/issues/407.

Original file: [15.txt](https://github.com/Sjors/nado-book/blob/d6e00aebc31164fbbcf5ad235b7874f282e0ff1c/transcripts/15.txt)

Added titles.
Cleaned Language fillers.

kouloumos commented:
Thank you @edilmedeiros! I added a missing speaker name and removed the `bitcoin-core` tag.
I'm also currently experimenting with using AI to create summaries and chapters for the transcripts, so I also added an AI-generated summary to this one, as well as a couple of extra chapters.

edilmedeiros commented:
> Thank you @edilmedeiros! I added a missing speaker name and removed the `bitcoin-core` tag. I'm also currently experimenting with using AI to create summaries and chapters for the transcripts, so I also added an AI-generated summary to this one, as well as a couple of extra chapters.

What tool are you using for the summaries?
